Source/Scene/Batched3DModel3DTileContent.js

```diff
@@ -154,6 +154,9 @@ define([
      * Part of the {@link Cesium3DTileContent} interface.
      */
     Batched3DModel3DTileContent.prototype.initialize = function(arrayBuffer, byteOffset) {
+        // Destroy expired resources if they exist.
+        destroyResources(this);
```

Comment:
So it might be a little more complicated than just that...

Anyway one benefit of the approach here is that it's not tied to the expire or unloading systems. You can just call `tile.requestContent` and it will automatically update the content. This is how it works right now; all the more complicated `unloadeExpired****` functions are only called when the request fails.

We can talk more tomorrow or after Siggraph.